ouroboros-consensus/src/Ouroboros/Storage/ImmutableDB/Util.hs

```diff
@@ -31,18 +33,36 @@ import           Ouroboros.Storage.ImmutableDB.Types
 renderFile :: String -> Epoch -> FsPath
 renderFile fileType epoch = [printf "%s-%03d.dat" fileType epoch]
 
-liftFsError :: Functor m => ExceptT FsError m a -> ExceptT ImmutableDBError m a
-liftFsError = withExceptT (UnexpectedError . FileSystemError)
-
-throwUserError :: (HasCallStack, Monad m)
-               => UserError -> ExceptT ImmutableDBError m a
-throwUserError ue = throwError $ UserError ue (popCallStack callStack)
--- 'popCallStack' to remove the call to 'throwError' from the call stack
-
-throwUnexpectedError :: Monad m
-                     => UnexpectedError -> ExceptT ImmutableDBError m a
-throwUnexpectedError ue = throwError $ UnexpectedError ue
-
+handleFsError :: ErrorHandling ImmutableDBError m
```

Comment:
This function is not used at the moment.

Reply:
Hmmm. Ok, will delete it. Odd :thinking: